content/projects/gsoc/students.adoc

```diff
@@ -1,7 +1,7 @@
 ---
 layout: project
-title: "Google Summer of Code. Information for students"
-description: "This page aggregates links to internal and external resources for Google Summer of Code students"
+title: "Google Summer of Code. Information for GSoC Contributors"
+description: "This page aggregates links to internal and external resources for Google Summer of Code Contributors"
```

Comment:
Since contributors is not a title, I think it should be lower case.
```suggestion
description: "This page aggregates links to internal and external resources for Google Summer of Code contributors"
```